front_end/panels/ai_chat/ui/SettingsDialog.ts

```
@@ -2319,6 +2421,459 @@ export class SettingsDialog {
+      try {
+        MCPRegistry.dispose();
+        updateMCPStatus();
+        updateDisconnectButton();
+        updateToolsList();
+      } catch (err) {
+        console.error('Failed to disconnect MCP:', err);
+      }
+    });
+    mcpDisconnectContainer.appendChild(mcpDisconnectButton);
+
+    const updateDisconnectButton = () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)